libs/category/testing/src/inmemory-backend/category.service.ts

```diff
@@ -55,5 +59,19 @@ export class DaffInMemoryBackendCategoryService implements InMemoryDbService {
         status: STATUS.OK
       };
     });
-  }
+	}
+
+	private generatePageSize(reqInfo) {
+		if(reqInfo.req.params.map && reqInfo.req.params.map.get('page_size') && reqInfo.req.params.map.get('page_size')[0]) {
+			return parseInt(reqInfo.req.params.map.get('page_size')[0], 10);
+		}
+		return Math.floor(Math.random() * 8 + 1);
```

Comment:
Can this be a constant?

Reply:
I made it be 10 unless the user passes one in